app/Token.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use LaravelFCM\Message\OptionsBuilder;
+use LaravelFCM\Message\PayloadDataBuilder;
+use LaravelFCM\Message\PayloadNotificationBuilder;
+use FCM;
+use Carbon\Carbon;
+
+class Token extends Model
+{
+    protected $table = 'tokens';
+
+    public function users()
+    {
+        return $this->hasMany(User::class, 'id', 'user_id');
+    }
+
+    public static function sendPhotoNotification($title, $body, $data, $groupArray)
```

Comment:
move to notification helper class, rename as sendNotification

Reply:
Better, in my opinion, I have everything in one model when I work with FCM.